style.json

```diff
@@ -4294,12 +4294,12 @@
         [
           "<=",
           "rank",
-          24
+          10
```

Comment:
See comment below.  
If the rank is modified in "poi-level-1", this value should be updated too.